client/devdocs/design/blocks.jsx

```diff
@@ -47,6 +47,7 @@ import PostStatus from 'blocks/post-status/docs/example';
 import ReaderAuthorLink from 'blocks/reader-author-link/docs/example';
 import ReaderSiteStreamLink from 'blocks/reader-site-stream-link/docs/example';
 import ReaderFullPostHeader from 'blocks/reader-full-post/docs/header-example';
+import ReaderSearchStreamHeader from 'reader/search-stream/docs/search-stream-header';
```

Comment:
I think we should choose one approach here:
- SearchStreamHeader is a block and moves into `blocks/`
- SearchStreamHeader is a standalone component and doesn't to into Devdocs at all

What do you think @samouri?